pkg/driver/driver_packet.go

```diff
@@ -172,3 +172,26 @@ func (d *PacketDriver) decodeMachineID(id string) string {
 	splitProviderID := strings.Split(id, "/")
 	return splitProviderID[len(splitProviderID)-1]
 }
+
+func packetTagsMapToString(tags map[string]string) []string {
+	ret := make([]string, 0)
+	for k, v := range tags {
+		ret = append(ret, fmt.Sprintf("%s:%s", k, v))
+	}
+	return ret
+}
+
+func packetTagsStringToMap(tags []string) map[string]string {
+	ret := map[string]string{}
```

Comment:
Is this used somewhere?

Reply:
This got added after rebasing. Its already there in master branch

Reply:
I think it has been again removed from the master by this https://github.com/gardener/machine-controller-manager/pull/273/files. Maybe we need to rebase again?